ui/admin/app/styles/app.scss

```diff
@@ -493,6 +493,13 @@
   }
 }
 
+// target worker filters
+.target-worker-filters {
+  .hds-form-field--layout-vertical {
+    margin-left: 2.5rem;
+  }
```

Comment:
Is there another class to select on which is more indicative of intent?

Reply:
Unfortunately, no, this is the only class name on the block. 

<img width="1215" alt="Screen Shot 2023-01-18 at 1 54 23 PM" src="https://user-images.githubusercontent.com/15043878/213303750-9e09eea9-c0f5-4aee-a5a6-3f18749443bf.png">